examples/getstarted/README.md

```diff
@@ -34,32 +34,28 @@ docker-compose -f docker-compose.dev.yml stop
 DB={dbName} yarn develop
 ```
 
-The way it works is that the `getstarted` app has a specific `database.js` config file that will use the `DB` environment variable to setup the right database connection. You can look at the code [here](./config/environments/development/database.js)
+The way it works is that the `getstarted` app has a specific `database.js` config file that will use the `DB` environment variable to setup the right database connection. You can look at the code [here](./config/database.js)
 
 **Warning**
 
 You might have some errors while connecting to the databases.
-They might be coming from a conflict between a locally running database instance and the docker instance. To avoid the errors either shutdown your local database instance or change the ports in the `./config/environments/development/database.js` and the `docker-compose.dev.yml` file.
+They might be coming from a conflict between a locally running database instance and the docker instance. To avoid the errors either shutdown your local database instance or change the ports in the `./config/database.js` and the `docker-compose.dev.yml` file.
 
 **Example**:
 
 `database.js`
 
 ```js
-module.exports = {
-  connections: {
-    default: {
-      connector: 'mongoose',
-      settings: {
-        // host: 'localhost',
-        // database: 'strapi',
-        // username: 'root',
-        // password: 'strapi',
-        port: 27099,
-      },
-      options: {},
-    },
+const mongo = {
```

Comment:
if you don't export your config it won't be used

Reply:
That's just a snippet of the code, if you see the structure now, they have all configs separated and export them together.

So in the end, this is exported =)

I understand your comment but maybe we just leave the lines were the person could edit or leave this snippet as an example and the person will see everything inside the file itself.

Reply:
Or would you prefer to revert this snippet and keep as it was before?

Reply:
If you can just reflect it with another db connection example and an export at the end of the file it would be perfect 👍